client/js/libs/handlebars/friendlydate.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+const moment = require("moment");
+
+module.exports = function(time) {
+	// See http://momentjs.com/docs/#/displaying/calendar-time/
+	return moment(new Date(time)).calendar(null, {
+		sameDay: "[Today]",
+		lastDay: "[Yesterday]",
+		lastWeek: "L", // Locale
```

Comment:
Actually, here's an idea to make this better.

Create a constant for this string and name it "Locale", or something, and then you don't need the comment. But still, my point remains.